webapp/graphql/PostMutations.js

```diff
@@ -4,27 +4,43 @@ export default () => {
   return {
     CreatePost: gql`
       mutation (
+        $id: ID
         $title: String!
+        $slug: String
         $content: String!
         $categoryIds: [ID]
-        $image: ImageInput
```

Comment:
Like i told on the other PRs this has been deleted somewhere in the process but is still needed